development/wasm-filter-plugins.md

```diff
@@ -59,31 +56,30 @@ Filters
   wasm                    WASM program filter
 ```
 
-## Build a WASM Filter for Filter Plugin
+## Build a Wasm filter for filter plugin
 
-Currently, Fluent Bit's WASM filter assumes C ABI that is also known as `wasm32-unknown-unknown` on Rust target and `wasm32-wasi` on TinyGo target.
+Currently, Fluent Bit's Wasm filter assumes C ABI, also known as `wasm32-unknown-unknown` on Rust target and `wasm32-wasi` on TinyGo target.
```

Comment:
```suggestion
The Fluent Bit Wasm filter assumes C ABI, also known as `wasm32-unknown-unknown` on Rust target and `wasm32-wasi` on TinyGo target.
```

Just to remove "Currently" and to remove the possessive because it's easy.